sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfig.java

```diff
@@ -76,6 +80,20 @@ public Double getTargetThroughputThreshold() {
      */
     public PriorityLevel getPriorityLevel() { return this.priorityLevel; }
 
+
+    /***
+     * Get the throughput bucket.
+     * <p>
+     * For more information about throughput bucket please visit
+     * <a href="https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/throughput-buckets?tabs=dotnet">Throughput buckets in Azure Cosmos DB</a>
```

Comment:
The doc should be extended for Java (min sdk versiona nd how to use the feature - would be better to refernece the java sample than .net - but can be done in separate PR)